homework.md

```diff
@@ -4,110 +4,110 @@
 
 What is 2 + 2?
 
-5
+4
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+White
 
 ## Question 3
 
 What is the  2 * 3 * 6 * 1 * 10 * 34?
 
-1322
+12,240
 
 ## Question 4 
 
 What does HTML stand for?
 
-Hyper Tall Massive Language
+HyperText Markup Language
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+Defines styles for web pages, including the design and layout appearance
 
 ## Question 6
 
 Using CSS, how would I select every `<h1>` tag on the page?
 
-```css
-#h1 {
+h1 { 
 
 }
-```
 
 ## Question 7
 
 What is JavaScript?
 
-A programming language used on the internet
```

Comment:
All programming languages are text-based even assembly used text for command which is interpreted to machine code which consists of 0s and 1s